app/javascript/components/Toolbar/index.js

```diff
@@ -94,33 +94,52 @@ const OfficeFilter = ({ value, onChange, offices }) => (
       underlineStyle={styles.underline}
       iconStyle={styles.icon}
     >
-      <MenuItem value="all" primaryText="All" style={styles.menuitem} />
+      <MenuItem value="all" primaryText={t('dashboard.layouteventstab.office.all')} style={styles.menuitem} />
       {offices.map((office, i) => (
         <MenuItem key={`office-${i}`} value={office.id} primaryText={office.name} style={styles.menuitem} />
       ))}
     </DropDownMenu>
   </div>
 )
 
-const Toolbar = (offices, showFilter, eventFilter, officeFilter, { label, view, views, onNavigate, onViewChange }) => (
+const toolbarTextMap = (t, key) => {
+  const map = {
+    month: t('calendar.bigcalendar.month'),
+    week: t('calendar.bigcalendar.week'),
+    work_week: t('calendar.bigcalendar.work_week'),
+    day: t('calendar.bigcalendar.day'),
+    agenda: t('calendar.bigcalendar.agenda'),
+  }
+  return map[key]
+}
+
+const Toolbar = (
+  offices,
+  showFilter,
+  eventFilter,
+  officeFilter,
+  t,
+  { label, view, views, onNavigate, onViewChange }
+) => (
   <div className={s.toolbar}>
     <div className={s.navBar}>
       <button className={s.todayBtn} type="button" onClick={() => onNavigate(navigate.TODAY)}>
-        Today
+        {t('dashboard.layoutdatetab.today')}
       </button>
       <button className={s.btn} type="button" onClick={() => onNavigate(navigate.PREVIOUS)}>
-        ‹
+        {t('dashboard.layoutdatetab.previous')}
       </button>
       <span className={s.label}>{label}</span>
       <button className={s.btn} type="button" onClick={() => onNavigate(navigate.NEXT)}>
-        ›
+        {t('dashboard.layoutdatetab.after')}
       </button>
     </div>
     <div className={s.filterBar}>
       <div className={s.filterDropdowns}>
-        <ShowFilter value={showFilter.value} onChange={(_e, _i, value) => showFilter.onChange(value)} />
-        <EventFilter value={eventFilter.value} onChange={(_e, _i, value) => eventFilter.onChange(value)} />
+        <ShowFilter t={t} value={showFilter.value} onChange={(_e, _i, value) => showFilter.onChange(value)} />
```

Comment:
`t` as a component property is not very descriptive. This makes it hard to quickly obtain context or understand how to set this value.